src/hotspot/share/prims/jvmtiExport.cpp

```diff
@@ -1506,6 +1506,8 @@ void JvmtiExport::post_resource_exhausted(jint resource_exhausted_flags, const c
 
   JavaThread *thread  = JavaThread::current();
 
+  log_error(os)("Resource Exhausted (%s)", description != nullptr ? description : "no info");
```

Comment:
Shouldn't that be `log_error(jvmti)`?
I'd also suggest the text "Posting Resource Exhausted event: %s" with "unknown" for a null description.

Reply:
Sure, no problem.